courseManagementService/src/main/java/com/lms/courseManagementService/mapper/ModuleMapper.java

```diff
@@ -0,0 +1,60 @@
+package com.lms.courseManagementService.mapper;
+
+import com.lms.courseManagementService.dto.LessonDTO;
+import com.lms.courseManagementService.dto.ModuleDTO;
+import com.lms.courseManagementService.model.Course;
+import com.lms.courseManagementService.model.Lesson;
+import com.lms.courseManagementService.model.Module;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ModuleMapper {
```

Comment:
Awesome implementation, you can also search for mapStruct to help you doing this mapping